pkg/webhook/util/filter.go

```diff
@@ -0,0 +1,25 @@
+package util
+
+import (
+	apierrors "k8s.io/apimachinery/pkg/api/errors"
+
+	ctlbackup "github.com/harvester/harvester/pkg/controller/master/backup"
+	ctlharvesterv1 "github.com/harvester/harvester/pkg/generated/controllers/harvesterhci.io/v1beta1"
+	"github.com/harvester/harvester/pkg/webhook/indexeres"
+)
+
+func HasInProgressingVMBackupBySourceUID(cache ctlharvesterv1.VirtualMachineBackupCache, sourceUID string) (bool, error) {
+	vmBackups, err := cache.GetByIndex(indexeres.VMBackupBySourceUIDIndex, sourceUID)
+	if err != nil {
+		if apierrors.IsNotFound(err) {
+			return false, nil
+		}
```

Comment:
NIT: Can safely pass `nil` into range keyword

https://go.dev/play/p/xG9O-VqIcl7

```suggestion
	if err != nil && !apierrors.IsNotFound(err) {
```

Reply:
Updated it. Thank you.